### PR TITLE
Add HTS iNet cask

### PR DESCRIPTION
**Pull request by jaingaurav**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

File: Casks/hts-inet.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'hts-inet' do
+  version '9.0.1'
+  sha256 '307ca070a0e36718378a90a4f5d8ce8a31b94a17dcc2e7f500630f480ca5ffaa'
+
+  url 'http://www.htstechsupport.com/HTS_iNet.dmg'
```
Comment (vitorgalvao) [inline Casks/hts-inet.rb]: Where on the page is the link to this download?

Comment (jaingaurav) [inline Casks/hts-inet.rb]: So they originally had the url links directly available at http://www.htstechsupport.com. But they are currently asking for a user id to get the link. I don't believe the link has changed in years.

Note, that the application does require you to login in order to operate.

Comment (jaingaurav) [inline Casks/hts-inet.rb]: Would you like me to update the link or use the product link page?

**Comment by jaingaurav** [comment]
So I updated the cask, but unfortunately the cask will be installed all the time now. Also, the download is pretty sizable. Is there any way to avoid re-installing the binary?

**Comment by jaingaurav** [comment]
Can cask use the application bundle version as a check? Or possibly a SHA-256 of the binary?

**Comment by vitorgalvao** [comment]
> but unfortunately the cask will be installed all the time now.

What do you mean? No, it won’t.

> Can cask use the application bundle version as a check? Or possibly a SHA-256 of the binary?

No, the check is done for integrity of the download.

File: Casks/hts-inet.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/hts-inet.rb]: Either you change the homepage to `http://www.htstechsupport.com` or [we cannot accept this here](https://github.com/caskroom/homebrew-cask/blob/7c010b42bd7ac679c1f002a0368b36bb3f495612/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment).

**Comment by jaingaurav** [comment]
Sorry, you're right, it doesn't get installed all the time. But when does brew upgrade to a new version? Will it only upgrade with the `--force` option then?

Also the travis build failure seems to be unrelated as it applies to Caffeine.app

**Comment by jaingaurav** [comment]
As requested, I have updated the url. Also seems the travis build worked this time around.

Thanks a lot for looking at this PR.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 03cf9f2ee0ffd9404dbfad21fdb2fa8139faf0c1 to keep commit history cleaner. Your contribution is still credited to you.

**Comment by vitorgalvao** [comment]
> But when does brew upgrade to a new version? Will it only upgrade with the `--force` option then?

Currently yes.

**Pull request closed without merging**